Tasks/SshV0/ssh.ts

```diff
@@ -198,8 +207,28 @@ function tryDeleteFile(filePath: string): void {
 run();
 
 function getReadyTimeoutVariable(): number {
-    let readyTimeoutString: string = tl.getInput('readyTimeout', true);
+    const readyTimeoutString: string = tl.getInput('readyTimeout', true);
     const readyTimeout: number = parseInt(readyTimeoutString, 10);
 
     return readyTimeout;
-} 
+}
+
+/**
+ * Read port number from SSH endpoint input
+ * @param {string} sshEndpoint - name of the service endpoint
+ * @returns {number} Port number of remote ssh server.
+ *
+ * If port is not specified in the endpoint configuration, the default port value number will be returned.
+ */
+function getServerPort(sshEndpoint: string): number {
+    let port: number = DEFAULT_SSH_PORT;
+    const portString: string = tl.getEndpointDataParameter(sshEndpoint, 'port', true);
+
+    if (portString) {
+        port = parseInt(portString, 10);
```

Comment:
Could you add validation for NaN values?

Reply:
Thanks for your comment! 
Validation for NaN values is not necessary since port number value validation handled by Azure itself.

![image](https://user-images.githubusercontent.com/14060121/86265616-a984da00-bbcc-11ea-9da5-d1b142798bba.png)